pkg/cmd/pr/shared/params.go

```diff
@@ -20,9 +20,10 @@ func WithPrAndIssueQueryParams(client *api.Client, baseRepo ghrepo.Interface, ba
 	if state.Title != "" {
 		q.Set("title", state.Title)
 	}
-	if state.Body != "" {
-		q.Set("body", state.Body)
-	}
+	// We always want to set body, even if it's empty, to prevent the web interface to apply the default
```

Comment:
A nitpick, but "...to prevent the web interface _from applying_ the default..." sounds better than "...to prevent the web interface _to apply_ the default...".